Nodejs/Product/Nodejs/Repl/InfoReplCommand.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using System.Diagnostics;
+using System.Globalization;
+using System.Threading.Tasks;
+
+namespace Microsoft.NodejsTools.Repl
+{
+    [Export(typeof(IReplCommand))]
+    internal class InfoReplCommand : IReplCommand
+    {
+        public Task<ExecutionResult> Execute(IReplWindow window, string arguments)
+        {
+            if (window.Evaluator is NodejsReplEvaluator nodeEval)
+            {
+                try
+                {
+                    var nodeExePath = nodeEval.NodeExePath;
+                    if (string.IsNullOrEmpty(nodeExePath))
+                    {
+                        nodeExePath = nodeEval.GetNodeExePath();
+                    }
+                    var nodeVersion = FileVersionInfo.GetVersionInfo(nodeExePath);
+
+                    window.WriteLine(string.Format(CultureInfo.CurrentUICulture, Resources.ReplNodeInfo, nodeExePath));
+                    window.WriteLine(string.Format(CultureInfo.CurrentUICulture, Resources.ReplNodeVersion, nodeVersion.ProductVersion));
+                }
+                catch(Exception e)
```

Comment:
Is this boilerplate or do we actually expect an exception?  If the main failure mode is a bad path, it seems like detecting it and printing a message to that effect would be more helpful than a stack trace.

Reply:
We already check when setting the path in a bunch of places. This is a backstop to not crash the interactive window when trying to get diagnostics.